R/wb_functions.R

```diff
@@ -639,6 +635,30 @@ wb_to_df <- function(
 
   }
 
+  # is.na needs convert
+  if (skipEmptyRows) {
+    empty <- vapply(seq_len(nrow(z)), function(x) all(is.na(z[x, ])), NA)
+
+    z  <- z[!empty, , drop = FALSE]
+    tt <- tt[!empty, , drop = FALSE]
+  }
+
+  if (skipEmptyCols) {
+
+    empty <- vapply(z, function(x) all(is.na(x)), NA)
+
+    if (any(empty)) {
+      sel <- which(names(empty) %in% names(empty[empty == TRUE]))
```

Comment:
I think this is the same as `which(empty)`?

Reply:
Ah yes thanks for spotting!